src/pages/zoeken/[slug]/[ie]/index.tsx

```diff
@@ -735,7 +752,7 @@ const ObjectDetailPage: NextPage<ObjectDetailPageProps> = ({ title, url }) => {
 					<div className="u-pb-24 p-object-detail__actions">
 						<div className="p-object-detail__primary-actions">
 							{showMetadataExportDropdown && renderExportDropdown()}
-							{canRequestMaterial && (
+							{isNotKiosk && (
```

Comment:
I would leave this as it is, but just as a general rule, we prefer to do user group specific showing/hiding of UI based on permissions and not based on user group.

This is because if meemoo at some point in the future would decide that they do want to show this also for kiosk users, it would require dev changes from us. While if this is checked off by a permission, they could change the permissions in the admin dashboard and the changes would happen automatically without needing dav changes.

We can convert this to a permission in v3